commit 362d4c355649de8d00a014b1f59be4478159e71b (refs/changes/99/5099/5)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2018-11-28T01:42:00+00:00 (11 months ago)
    
    TRITON-779 store affinity rules when provisioning
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/lib/metadata.js b/lib/metadata.js
index 7ca7744..3b2de23 100644
--- a/lib/metadata.js
+++ b/lib/metadata.js
@@ -170,10 +170,10 @@ function list(req, res, next) {
                 }
 
                 if (Object.keys(md2).length) {
+                    delete md2.locality;
                     Object.keys(md2).forEach(function (k) {
                         if (/_pw$/.test(k)) {
-                            md2[k.replace(/_pw$/, '')] =
-                                md2[k];
+                            md2[k.replace(/_pw$/, '')] = md2[k];
                             delete md2[k];
                         }
                     });
@@ -220,10 +220,10 @@ function get(req, res, next) {
             } catch (e) {}
 
             if (Object.keys(md).length) {
+                delete md.locality;
                 Object.keys(md).forEach(function (k) {
                     if (/_pw$/.test(k)) {
-                        md[k.replace(/_pw$/, '')] =
-                            md[k];
+                        md[k.replace(/_pw$/, '')] = md[k];
                         delete md[k];
                     }
                 });
diff --git a/package.json b/package.json
index a5bd86a..be3362f 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "cloudapi",
     "description": "SmartDataCenter CloudAPI",
-    "version": "9.3.0",
+    "version": "9.3.1",
     "author": "Joyent (joyent.com)",
     "private": true,
     "engines": {
